.prettierrc.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I suggest to synchronize these rules with BE [ones](https://github.com/Code-the-Dream-School/ii-practicum-team-2-back/blob/main/.prettierrc) to make both repos more consistent.